Travis intermittently fails

Travis now seems to periodically fail for PhantomJS downloading (use a Bitflux bundle?)

```

> phantomjs-prebuilt@2.1.4 install /home/travis/build/ScottLogic/StockFlux/node_modules/BitFlux/node_modules/phantomjs-prebuilt
> node install.js
Found PhantomJS at /usr/local/phantomjs/bin/phantomjs ...verifying
PhantomJS detected, but wrong version 1.9.8 @ /usr/local/phantomjs/bin/phantomjs.
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2
Saving to /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Receiving...
Error requesting archive.
Status: 403
Request options: {
  "uri": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2",
  "encoding": null,
  "followRedirect": true,
  "headers": {
    "User-Agent": "npm/2.14.2 node/v4.0.0 linux x64"
  },
  "strictSSL": true
}
Response headers: {
  "x-amz-request-id": "A7CB1D0FB7589B38",
  "x-amz-id-2": "fDCqVbiB7y8xrtemqhgk2uS+UDJROYUTQUSA6J5IyaXdFjOJFos3whSNatOB+qM8LGyueTy4m+0=",
  "content-type": "application/xml",
  "transfer-encoding": "chunked",
  "date": "Wed, 23 Mar 2016 12:45:54 GMT",
  "server": "AmazonS3",
  "connection": "close"
}
Make sure your network and proxy settings are correct.
If you continue to have issues, please report this full log at https://github.com/Medium/phantomjs
> phantomjs@1.9.19 install /home/travis/build/ScottLogic/StockFlux/node_modules/BitFlux/node_modules/grunt-svg-sprite/node_modules/svg-sprite/node_modules/phantomjs
> node install.js
Found PhantomJS at /usr/local/phantomjs/bin/phantomjs ...verifying
Writing location.js file
PhantomJS is already installed on PATH at /usr/local/phantomjs/bin/phantomjs
npm WARN engine agentkeepalive@0.2.4: wanted: {"node":"0.10.x"} (current: {"node":"4.0.0","npm":"2.14.2"})
npm ERR! Linux 3.13.0-40-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v4.0.0/bin/node" "/home/travis/.nvm/versions/node/v4.0.0/bin/npm" "install"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE
npm ERR! phantomjs-prebuilt@2.1.4 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the phantomjs-prebuilt@2.1.4 install script 'node install.js'.
npm ERR! This is most likely a problem with the phantomjs-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls phantomjs-prebuilt
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/ScottLogic/StockFlux/npm-debug.log
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,9 +6,10 @@ node_js:
 sudo: false
 
 install:
-- npm install
-- npm install -g grunt-cli
 - npm install -g npm@'>=2.13.5'
+- npm install -g grunt-cli
+- npm install -g phantomjs-prebuilt --phantomjs_cdnurl=http://cnpmjs.org/downloads
+- npm install
 
 env:
   global:

